testing/beacon-api/tests/serialization.rs

```diff
@@ -0,0 +1,67 @@
+use std::{fs, io::Read};
+
+use alloy_primitives::b256;
+use ream_consensus::deneb::{beacon_block::SignedBeaconBlock, beacon_state::BeaconState};
+use ream_rpc::types::response::BeaconVersionedResponse;
+use serde_json::Value;
+
+#[tokio::test]
+async fn test_beacon_state_serialization() -> anyhow::Result<()> {
+    pub type Response = BeaconVersionedResponse<BeaconState>;
+
+    let file_path = "./tests/assets/state.json";
+    let original_json = read_json_file(file_path)?;
+    println!("Serialization initiated");
```

Comment:
```suggestion
#[tokio::test]
async fn test_beacon_state_serialization() -> anyhow::Result<()> {
    pub type Response = BeaconVersionedResponse<BeaconState>;

    let original_json = read_json_file("state.json")?;
```

Can `./tests/assets/` be a const, and you pass in the extended file name + path